solidity/contracts/converter/types/liquidity-pool-v2/LiquidityPoolV2Converter.sol

```diff
@@ -482,11 +482,38 @@ contract LiquidityPoolV2Converter is LiquidityPoolConverter {
     function doConvert(IERC20Token _sourceToken, IERC20Token _targetToken, uint256 _amount) private returns (uint256, uint256, Fraction memory) {

```

Comment:
I guess that's the new `prepareConversion`...

Reply:
In progress...